index.js

```diff
@@ -10,11 +10,10 @@ app.use(jsonParser)
 
 app.put('/notes/:id', (req, res) => {
   const noteId = parseInt(req.params.id, 10)
-  if (req.body.content) {
-    notes[noteId - 1].content = req.body.content
-  }
-  if (req.body.title) {
-    notes[noteId - 1].title = req.body.title
+  for (const key in req.body) {
+    if (key === 'title' || key === 'content') {
```

Comment:
No, what if there's another key on notes tomorrow? Will you just keep adding `||`s?